docs/Project-Configuration/Github-Integrations.md

```diff
@@ -109,7 +112,13 @@ buildvariants:
     tasks:
       - name: "print_apple"
         create_check_run:
-          path_to_outputs: "src/output.json" ## for no output use ""
+          path_to_outputs: "src/output.json" ## this file will be checked after the task finishes running
+      - name: "valid_check_run"
+        create_check_run:
+          path_to_outputs: "" ## Evergreen will create a default output if a path is not specified
```

Comment:
can we explain what default output looks like here or link to documentation that explains it?